posthog/utils.py

```diff
@@ -303,6 +304,19 @@ def get_default_event_name():
     return "$pageview"
 
 
+def get_active_plugins(team_id: int) -> List[int]:
```

Comment:
We've already got `posthog/plugins/utils.py`, this could fit in pretty well there